README.md

```diff
@@ -86,6 +86,7 @@ You can pass props to the plugin config object to configure:
 | `iPhoneDeploymentTarget` | optional | Target `IPHONEOS_DEPLOYMENT_TARGET` value to be used when adding the iOS [NSE](https://documentation.onesignal.com/docs/service-extensions). A deployment target is nothing more than the minimum version of the operating system the application can run on. This value should match the value in your Podfile e.g: `"12.0"`. |
 | `smallIcons`             | optional | An array of local paths to small notification icons for Android.  Image should be white, transparent, and 96x96 in size.  Input images will be automatically scaled down and placed in the appropriate resource folders.  e.g: `["./assets/ic_stat_onesignal_default.png"]`. See https://documentation.onesignal.com/docs/customize-notification-icons#small-notification-icons.  |
 | `largeIcons`             | optional | An array of local paths to large notification icons for Android.  Image should be white, transparent, and 256x256 in size.  e.g: `["./assets/ic_onesignal_large_icon_default.png"]`. See https://documentation.onesignal.com/docs/customize-notification-icons#large-notification-icons.                                                                                                |
+| `nse`                    | optional | The local path to a custom Notification Service Extension (NSE). The NSE will typically start as a copy of the [default NSE](https://github.com/OneSignal/onesignal-expo-plugin/blob/main/support/serviceExtensionFiles/NotificationService.m), then altered to support any custom logic required.|
```

Comment:
`nse` is ambiguous for a `string` type. This should be an option like `nse: { path: "" }` or specific like `nse_main_source_file_path`.

Also maybe include `ios` in the field name, since OneSignal also has an Android NSE.

Also recommend providing an example like some of the other fields above.

Reply:
@jkasten2 updated to be `iosNSEFilePath`. Thanks for the comment!